build.gradle

```diff
@@ -25,7 +25,7 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "com.google.android.exoplayer:exoplayer:2.13.3"
+    implementation "com.google.android.exoplayer:exoplayer:2.13.2"
```

Comment:
is this change relevant? maybe we can change this in another PR instead

Reply:
this happened because "fix-touch" was our main branch for quite some time. I can cherry-pick this to master before merge to keep the diff clean